src/vault/client/http.clj

```diff
@@ -130,6 +132,22 @@
   (reset! (:auth client) {:client-token (str/trim token)}))
 
 
+(defn- authenticate-wrap-token!
+  "Updates the token ref by making an unwrap request that returns the auth token."
+  [client credentials]
+  (let [{:keys [username password]} credentials]
+    (api-auth!
+      (str "wrapped token " credentials)
```

Comment:
I updated role-id logging to log a hash, but just removed the credential here since its not a stable identifier you'd want to match up with logs most likely.

Reply:
Actually the vault audit logs _do_ give you the SHA256 of the wrapping token in the response, so it could be useful for matching up later.